tests/test_verbs.py

```diff
@@ -119,7 +138,59 @@ def test_identify(app):
         assert len(granularity) == 1
         description = tree.xpath('/x:OAI-PMH/x:Identify/x:description',
                                  namespaces=NAMESPACES)
-        assert len(description) == 2
+
+        friends_element = description[0]
+        for element in friends_element.getchildren():
+            for child in element.getchildren():
+                assert child.tag == \
+                    '{http://www.openarchives.org/OAI/2.0/friends/}baseURL'
+
+        eprints_root = description[1]
+        children = eprints_root[0].getchildren()
+        assert children[0].tag == \
+            '{http://www.openarchives.org/OAI/2.0/eprints}content'
+        leaves = children[0].getchildren()
+        assert len(leaves) == 1
+        assert leaves[0].tag == \
+            '{http://www.openarchives.org/OAI/2.0/eprints}URL'
```

Comment:
check also the elements ``text`` properties as I mentioned in previous comment...
```
assert leaves[0].text == 'http://arXiv.org/arXiv_content.htm'
```